ui/app/templates/jobs/run/templates/new.hbs

```diff
@@ -0,0 +1,34 @@
+<section class="section">
+    <form class="new-job-template" autocomplete="off">
+        <div class={{if this.namespaceOptions "path-namespace"}}>
+            <label>
+            <span>
+                Template name
+            </span>
+            <Input
+                @type="text"
+                @value={{mut this.templateName}}
+                placeholder="your-template-name-here"
+                class="input path-input"
+                {{autofocus}}
+                data-test-template-name
+            />
+            </label>
```

Comment:
I'm not sure we want this here; we should instead add it into the VariableForm::JobTemplateEditor. Name is already a field there.

Reply:
OK! I'm following the pattern in `variable-form.hbs`, where `path` isn't included in the `variable-form.job-template-editor` component.

Do you want me to create this as a conditionally rendered part of the template in that component?